docs/includes/config/io.helidon.webclient.Proxy.adoc

```diff
@@ -0,0 +1,60 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:description: Configuration of io.helidon.webclient.Proxy
+:keywords: helidon, config, io.helidon.webclient.Proxy
+:basic-table-intro: The table below lists the configuration keys that configure io.helidon.webclient.Proxy
+
+[source,text]
+.Type
+----
+io.helidon.webclient.Proxy
+----
+
+
+
+==== Configuration options
+
+
+
+
+Optional configuration options:
+[cols="3,3,2,5"]
+
+|===
+|key |type |default value |description
+
+|`password` |string |{nbsp} |Sets a new password for the proxy.
+|`port` |int |{nbsp} |Sets a port value.
+|`host` |string |{nbsp} |Sets a new host value.
+|`use-system-selector` |boolean |`false` |Configure proxy from environment variables and system properties.
+|`username` |string |{nbsp} |Sets a new username for the proxy.
+|`type` |string (HTTP, SOCKS_4, SOCKS_5, SYSTEM) |`HTTP` |Sets a new proxy type.
+|`no-proxy` |string[&#93; |{nbsp} |Configure a host pattern that is not going through a proxy.
+ <p>
+ Options are:
+ <ul>
```

Comment:
This is not correct. We need to update our generator to create a better table for this (https://github.com/tomas-langer/config-documentation/pull/2).